python/mxnet/kvstore.py

```diff
@@ -379,12 +400,33 @@ def _send_command_to_servers(self, head, body):
 def create(name='local'):
     """Creates a new KVStore.
 
+    For single machine training, there are two commonly used types:
+
+    ``local``: all gradients are copied to CPU memory and weights are updated there.
```

Comment:
more succinct to say:

"Copies all gradients to CPU memory and updates weights there."